components/Profile/Staking.jsx

```diff
@@ -165,6 +166,14 @@ const StakingDetails = ({ profile }) => {
 
       // First unstake
       await unstake({ account });
+      // Service is now transferred back to the user,
+      // In order to stake again, need to approve the transfer
+      // to contribute manager
+      await approveServiceTransfer({
```

Comment:
here I approve service transfer to contribute manager, please check if it's correct @kupermind